nbconvert/templates/html/basic.tpl

```diff
@@ -209,3 +209,31 @@ var element = $('#{{ div_id }}');
 </script>
 </div>
 {%- endblock -%}
+
+{%- block data_widget_state scoped %}
+{% set div_id = uuid4() %}
+{% set content = output.data['application/vnd.jupyter.widget-state+json'] %}
+<div id="{{ div_id }}"></div>
+<div class="output_subarea output_widget_state {{extra_class}}">
+<script type="tapplication/vnd.jupyter.widget-state+json">
+var element = $('#{{ div_id }}');
+{
+"model_id": "{{content.model_id}}"
+}
+</script>
```

Comment:
I have an example of notebook with these new mime types here: https://gist.github.com/SylvainCorlay/d6d8cd44e032523d07e102f5dc0bb1df

A bunch of output areas have content of mime type
```
"application/vnd.jupyter.widget-view+json": {
       "model_id": "b59af7d27c594924957e1aa26257751f"
      }
```